.github/CODEOWNERS

```diff
@@ -12,60 +12,105 @@ yarn.lock                                                               @backsta
+/workspaces/repo-tools                                                  @backstage/community-plugins-maintainers
 /workspaces/report-portal                                               @backstage/community-plugins-maintainers  @yashoswalyo @deshmukhmayur @riginoommen
 /workspaces/rollbar                                                     @backstage/community-plugins-maintainers  @andrewthauer
 /workspaces/scaffolder-backend-module-annotator                         @backstage/community-plugins-maintainers  @BethGriggs @debsmita1
 /workspaces/scaffolder-backend-module-kubernetes                        @backstage/community-plugins-maintainers  @BethGriggs @debsmita1
-/workspaces/scaffolder-backend-module-sonarqube                         @backstage/community-plugins-maintainers  @04kash @schultzp2020
-/workspaces/scaffolder-backend-module-servicenow                        @backstage/community-plugins-maintainers  @schultzp2020
 /workspaces/scaffolder-backend-module-regex                             @backstage/community-plugins-maintainers  @04kash
+/workspaces/scaffolder-backend-module-servicenow                        @backstage/community-plugins-maintainers  @schultzp2020
+/workspaces/scaffolder-backend-module-sonarqube                         @backstage/community-plugins-maintainers  @04kash @schultzp2020
 /workspaces/scaffolder-relation-processor                               @backstage/community-plugins-maintainers  @04kash
+/workspaces/sentry                                                      @backstage/community-plugins-maintainers
+/workspaces/shortcuts                                                   @backstage/community-plugins-maintainers
 /workspaces/sonarqube                                                   @backstage/community-plugins-maintainers  @backstage/sda-se-reviewers
+/workspaces/splunk                                                      @backstage/community-plugins-maintainers
+/workspaces/stack-overflow                                              @backstage/community-plugins-maintainers
+/workspaces/stackstorm                                                  @backstage/community-plugins-maintainers
 /workspaces/tech-insights                                               @backstage/community-plugins-maintainers  @xantier @punkle
+/workspaces/tech-radar                                                  @backstage/community-plugins-maintainers
 /workspaces/tekton                                                      @backstage/community-plugins-maintainers  @caugello @cryptorodeo
+/workspaces/todo                                                        @backstage/community-plugins-maintainers
 /workspaces/topology                                                    @backstage/community-plugins-maintainers  @christoph-jerolimov @ciiay @debsmita1 @divyanshiGupta @its-mitesh-kumar @logonoff
-/workspaces/multi-source-security-viewer                                @backstage/community-plugins-maintainers  @caugello @cryptorodeo
```

Comment:
This exists here: https://github.com/backstage/community-plugins/tree/main/workspaces/multi-source-security-viewer

Reply:
And those care the correct owners

Reply:
Oups, mea culpa: missed this in the alpha-order somewhere.

Reply:
No worries, just wanted to make sure we kept them in there 👍